src/flux/task.coffee

```diff
@@ -221,6 +221,13 @@ TaskConfig =
 
       result
 
+    getExternalStep: (id) ->
+      step = _.findWhere(@_steps[id], type: 'external_url')
+      unless step?.external_url.match(/^https?:\/\//)
```

Comment:
Yep, this is her implementation, I've just moved it to the store so it can be shared between the "normal" click and the "right-click" components